derp/derp_server.go

```diff
@@ -103,6 +104,12 @@ type Server struct {
 	sentTo map[key.Public]map[key.Public]int64 // src => dst => dst's latest sclient.connNum
 }
 
+var expvarVersion expvar.String
```

Comment:
just put this and the init down in the ExpVar method?

Reply:
Done.